src/constants.ts

```diff
@@ -8,18 +6,7 @@ export const API_BASE_MAINNET = "https://api.opensea.io";
 export const API_BASE_TESTNET = "https://testnets-api.opensea.io";
 export const API_PATH = `/api/v${API_VERSION}`;
 
-export const MERKLE_VALIDATOR_MAINNET =
-  "0xbaf2127b49fc93cbca6269fade0f7f31df4c88a7";
-
-export const WETH_ADDRESS_BY_NETWORK = {
-  [Chain.Mainnet]: "0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2",
-  [Chain.Goerli]: "0xb4fbf271143f4fbf7b91a5ded31805e42b2208d6",
-} as const;
-
-export const DEFAULT_ZONE_BY_NETWORK = {
-  [Chain.Mainnet]: "0x0000000000000000000000000000000000000000",
-  [Chain.Goerli]: "0x0000000000000000000000000000000000000000",
-} as const;
+export const DEFAULT_ZONE = "0x0000000000000000000000000000000000000000";
```

Comment:
Do we want the zero address to be the default? If we do, should it be for all of them?

Reply:
you can use `ethers.constants.AddressZero`